index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Portfolio</title>
+</head>
+<body>
+<header class="toolbar">
+  <a href="#" class="toolbar logo"><img src="images/logo.png" alt=""></a>
+  <nav class="toolbar menu"><img src="images/menu.png" alt=""></nav>
+</header>
+<section class ="container">
+  <div class="bg-image1"></div>
+  <div class="bg-image2"></div>
+  <div class="bg-image3"></div>
+  <div class="bg-image4"></div>
+  <div class="bg-image6"></div>
+  <div class="bg-image7"></div>
+  <div class="bg-imagea"></div>
+  <div class="bg-imageb"></div>
+     <div class="container intro"><h1>I'm Victor,
+       Glad to see you!</h1></div>
+          <div class="container content">
+           <p>Hello! I am a software developer! I can help you build a product, feature or website.
+           Taka a look of my works.If you like what you see and have a project you need coded, don’t hesitate and contact me</p>
+          </div>
+     <ul class="container social">
+       <li><img src="images/connect.png" alt=""></li>
+     </ul>
+     <ul>
+       <li class="container linked"><img src="images/linkedin.png" alt=""></li>
+       <li class="container twit"><img src="images/twitter.svg" alt=""></li>
+       <li class="container vectorm"><img src="images/vectorm.svg" alt=""></li>
+       <li class="container vectorp"><img src="images/vectorp.svg" alt=""></li>
+       <li class="container vector"><img src="images/vector.svg" alt=""></li>
+     </ul>  
```

Comment:
According to the evaluation guidelines, each icon should be wrapped inside a link tag.

<img width="607" alt="Screen Shot 2022-05-26 at 15 34 48" src="https://user-images.githubusercontent.com/82426682/170575617-c73f3bed-9ad1-4547-a006-71af13d4edc1.png">

Kindly fix this.